data/rulei/rulei.txt

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2022 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Relic of the Past"
+	invisible
+	landing
+	to complete
+		never
+	npc save
+		government "Rulei"
+		personality heroic vindictive staying mute unconstrained frugal
+		system "Kanguwa"
+		ship "vyu-Ira" "Solemnity"
+
+mission "Ira Contact"
+	invisible
+	landing
+	to complete
+		never
+	on enter Kanguwa
+		fail
+		log "Factions" "Rulei" `The Rulei were likely an ancient, technologically advanced civilization from long ago.`
+			`They communicate through some sort of invasive psychonic ability.`
+		conversation
+			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. You feel an overwhelming sense of emptiness, but at the same time the intensity of this feeling suggests that something must be there. You barely have enough time to try and make sense of this oddity as only seconds later you start to hear ominous whispering, as if something is behind you.`
```

Comment:
```suggestion
			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. You feel an overwhelming sense of emptiness, but at the same time the intensity of this feeling suggests that something must be there. You have barely any time to try and make sense of this oddity, though; only seconds later you start to hear ominous whispering, as if something is behind you.`
```
That last sentence and the double "as" is feeling pretty awkward. Attempted to resolve with smallest possible change, but a more extensive one may be better.

Reply:
Yeah, I think maybe I should try and expand this a bit more rather than try and keep it shorter and change it only for the sake of the double "as". I'll think on it.